app/controllers/frontend_controller.rb

```diff
@@ -108,25 +104,25 @@ def analytics
   #
   def pick_buyer_or_provider_layout
     if site_account.master?
-      current_account ? 'provider' : 'provider/login'
+      current_account ? "provider" : "provider/login"
     else
       DEFAULT_LIQUID_LAYOUT
     end
   end
 
   def find_service(id = params[:service_id])
     services = if current_account.try!(:provider?)
-      current_account.accessible_services
+                 current_account.accessible_services
                else
-      site_account.accessible_services
-    end
+                 site_account.accessible_services
+               end
 
     @service = id.present? ? services.find(id) : services.default
   end
 
   def ensure_provider
     unless current_account.provider?
-      render_error 'Not authorized', :status => :not_authorized
+      render_error "Not authorized", :status => :not_authorized
```

Comment:
```suggestion
      render_error 'Not authorized', :status => :not_authorized
```

Reply:
Since we are refactoring this anyway:
```ruby
def ensure_provider
  return true if current_account.provider?
  render_error 'Not authorized', status: :not_authorized
  false
end
```